algorithms/src/main/kotlin/io/kinference/algorithms/gec/postprocessing/GecPostprocessor.kt

```diff
@@ -0,0 +1,40 @@
+package io.kinference.algorithms.gec.postprocessing
+
+import io.kinference.algorithms.gec.utils.SentenceCorrections
+import io.kinference.algorithms.gec.utils.TextCorrection
+
+/**
+ * function which create new sentence from incorrect sentence and TextCorrection list
+ * @param incorrectSentence - original sentence
+ * @param corrections - corrections of sentence
+ */
+fun transformSentence(incorrectSentence: String, corrections: List<TextCorrection>): String {
```

Comment:
In what form does it transform sentence? I mean, if it corrects sentences, then let's call it `correct`. Why transform?